src/27-ipfs-at-the-vatican/index.md

```diff
@@ -0,0 +1,158 @@
+
+<iframe src="https://ipfs.io/ipfs/Qmb7N1ZmoE9k2Yz6r4cDmMYQB64Gi1X2damE2FFA66LX4X/uv/uv.html#?manifest=https://ipfs.io/ipfs/QmYomyCpT1vKNovYvTjAStYq31JoaH8xdivjmWcDj5mpM1&c=0&m=0&s=0&cv=0&config=examples-config.json&locales=en-GB:English (GB),cy-GB:Cymraeg&xywh=-391,0,4780,2999&r=0" width="560" height="420" allowfullscreen frameborder="0"></iframe>
+
+
+If you look at the URIs in all of these examples, you'll see they all rely on the `https://ipfs.io` domain. These are _gateway links_, which make IPFS content backwards-compatible with the classical HTTP web. Every IPFS node is able to present itself as an HTTP server, acting as a _gateway_ between the HTTP web and the distributed peer-to-peer web. When one of these gateways receives a request, it uses the IPFS protocol to retrieve the corresponding content from the peer-to-peer web and then returns it over HTTP.
+
+If you use one of these gateway links in a web browser that doesn't speak IPFS, the content will still load because you're falling relying on the gateway to translate the requests for you. Over time, as browsers add support for IPFS (starting with the IPFS web extensions you can install in your browser) the browser will recognize those gateway requests, skip the HTTP connection, and instead resolve the content via IPFS.
+
+In the meantime, to confirm that you're actually retrieving content that any IPFS node could resolve, you can replace the ipfs.io portion of any url with the hostname of any IPFS gateway. There is a [thread in the IPFS discussion forum](https://discuss.ipfs.io/t/curated-list-of-ipfs-gateways/620) where people are gathering a list of gateways you could test out.
+
+## The Second Demo: Serverless Collaborative Annotations on IPFS
+
+Our second demo is more involved and it requires more explanation. The demo focuses on allowing people to annotate images and sync those annotations across the peer-to-peer network without relying on an annotation server. To support this real-time collaborative sharing of data we use [CRDTs](https://en.wikipedia.org/wiki/Conflict-free_replicated_data_type) and [IPFS pubsub](../25-pubsub/).
+
+One obstacle to enabling annotations of IIIF content is the fact that nobody wants to run an annotation server. Institutions want to let their own patrons and curators create annotations. Though they are also interested in seeing the annotations other people create, they don't want to get stuck storing the whole world's annotations indefinitely. That would be like your local public library committing to store all the notes anyone ever made about the books in their collection rather than letting everyone take their own notes home with them and store those notes as they see fit.
```

Comment:
> That would be like your local public library committing to store all the notes anyone ever made about the books in their collection rather than letting everyone take their own notes home with them and store those notes as they see fit.

This sentence is pretty clunky. Breaking it up will simplify the subject/possesive pairs (who's each of these theys?).